spec/jobs/rapid_ready_for_decision/form526_asthma_job_spec.rb

```diff
@@ -11,14 +11,7 @@
   end
 
   let(:user) { FactoryBot.create(:disabilities_compensation_user, icn: '2000163') }
-  let(:auth_headers) do
-    EVSS::DisabilityCompensationAuthHeaders.new(user).add_headers(EVSS::AuthHeaders.new(user).to_h)
-  end
-  let(:submission) do
-    create(:form526_submission, :with_uploads,
-           user_uuid: user.uuid,
-           auth_headers_json: auth_headers.to_json)
-  end
+  let(:submission) { create(:form526_submission, :with_uploads, user: user) }
```

Comment:
Unrelated change: this test setup simplification was suggested by @yoomlam a few weeks ago. Essentially, passing `user: user` to the factory takes care of all the rest.

Reply:
Nice! look at all those deleted lines

Reply:
I wonder if you even need `:with_uploads`.